### ISSUE TITLE
Webhook for products added to promotion

### ISSUE THREAD

**Issue by timuric** [saleor/saleor#14821]
## Problem
When an item is added to a promotion, it is necessary to update the price in external integration.

Use case:
Admin adds an item to promotion, which changes its price on PDP but not on the listing page that is powered by a search engine.
When the item is added to the catalog promotion, the webhook could provide product lists that are added or removed

**Comment by aniav** [comment]
When we approach this topic it would be good to think what is the right place to address it. Maybe we could use product related webhook instead of a promotions based one? The idea is that downstream systems need to know about the most up to date price of the products but these prices will sometimes come from a promotion apply and sometimes from a price change through our API.

**Comment by timuric** [comment]
@aniav agree; firing product related webhook is more intuitive and universal

**Comment by julienmarie** [comment]
Any update on that?
